src/RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -107,6 +107,8 @@ private void RecordTransferHistory(StoreView snapshot, UInt160 scriptHash, UInt1
             var eventName = stateItems[0].GetString();
             if (eventName != "Transfer") return;
             if (stateItems.Count < 4) return;
+            if (stateItems[1] is VM.Types.Null) stateItems[1] = null;
```

Comment:
Better to do this conversion right on the beginning instead of using VM.Types.Null?

Reply:
I think it is not necessary to convert it to null. We can check it by calling `stateItems[1].IsNull`.

Reply:
yes, I have removed it.